dev/KairaGupta/importance_score/importance_score.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+from sklearn import metrics
+import matplotlib.pyplot as plt
+
+def imp_score_singular_datapoints(model, train_features, train_labels, test_features, test_labels, i_start, i_end):
+    """
+        Inputs: 
+            model: model object which must have function model.fit(train_features, train_labels)
+            train_features: numpy array of feature values for training
+            train_labels: numpy array of correct labels for train_features
+            test_features: numpy array of feature values for test data
+            test_labels: numpy array of correct labels for test data
+            i_start: start index to remove features from train_labels (one-by-one)
+            i_end: end index (not inclusive) to remove features from train_labels (one-by-one)
+        Output:
+            Plot of difference in performance scores (accuracy and f-score) on the test set by removing data points (one-by-one) from training data.
+    """
+    
+    fs_list = []
+    acc_list = []
+    
+    temp_model = model
```

Comment:
nit: full model (or rather model trained with full data availability)
resulting in: `fs_full_data` and `acc_full_data` 
could have been trained inside the `for` loop resulting in less code clutter.

Not by any means a critical change, just a note on code cleanliness.

Reply:
Sure! I will make these changes ASAP! Thanks